File: php/pertemuan10/atm.php
<?php

$nama = readline("Masukkan Nama Anda: ");
$benar = readline("Apakah Anda Yakin Ingin Melanjutkan Transaksi? (Y/N): ");

$saldoAwal = 0;

if ($benar == "Y" || $benar == "y") {
    echo "=====================================================\n";
    echo "Selamat Datang Di ATM, $nama\n";
    echo "Saldo Anda Saat Ini: Rp $saldoAwal\n";
    echo "=====================================================\n";
} else {
    echo "=====================================================\n";
    echo "          Terima Kasih Atas Kunjungan Anda\n";
    echo "=====================================================\n";
    exit;
}

$inputSaldo = readline("Masukkan Saldo : ");

echo "=====================================================\n";
echo "NAMA ANDA              : $nama\n";
echo "SALDO ANDA SEKARANG    : Rp $inputSaldo\n";
echo "=====================================================\n\n";

echo "===================== MENU ATM ======================\n";
echo "1. CEK SALDO\n";
echo "2. TARIK TUNAI\n";
echo "3. SETOR TUNAI\n";
echo "4. TRANSFER\n";
echo "5. KELUAR\n";
echo "=====================================================\n";

$menu = readline("Pilih Menu : ");

switch ($menu) {
    case 1:
        echo "=====================================================\n";
        echo "Saldo Anda Saat Ini: Rp $inputSaldo\n";
        echo "=====================================================\n";
        break;

    case 2:
        echo "Saldo Anda Saat Ini: Rp $inputSaldo\n";
        $tarik = readline("Masukkan Jumlah Tarik Tunai: ");
        echo "=====================================================\n";
        echo "Saldo Anda Saat Ini: Rp " . ($inputSaldo - $tarik) . "\n";
        echo "=====================================================\n";
        break;

    case 3:
        echo "Saldo Anda Saat Ini: Rp $inputSaldo\n";
        $setor = readline("Masukkan Jumlah Setor Tunai: ");
        echo "=====================================================\n";
        echo "Saldo Anda Saat Ini: Rp " . ($inputSaldo + $setor) . "\n";
        echo "=====================================================\n";
        break;

    case 4:
        echo "Saldo Anda Saat Ini: Rp $inputSaldo\n";
        $tf = readline("Masukkan Jumlah Transfer: ");
        echo "=====================================================\n";
        echo "Saldo Anda Saat Ini: Rp " . ($inputSaldo - $tf) . "\n";
        echo "=====================================================\n";
        break;

    case 5:
        echo "=====================================================\n";
        echo "          Terima Kasih Atas Kunjungan Anda\n";
        echo "=====================================================\n";
        break;

    default:
        echo "=====================================================\n";
        echo "Menu Tidak Tersedia\n";
        echo "=====================================================\n";
        break;
}
?>

File: python/pertemuan10/atm.py
import os

nama = input("Masukkan Nama Anda: ")
benar = input("Apakah Anda Yakin Ingin Melanjutkan Transaksi? (Y/N): ")

saldo_awal = 0

print("=====================================================")

if benar.lower() == "y":
    os.system("cls" if os.name == "nt" else "clear")
    print("=====================================================")
    print(f"Selamat Datang Di ATM, {nama}")
    print(f"Saldo Anda Saat Ini: Rp {saldo_awal}")
    print("=====================================================")
else:
    os.system("cls" if os.name == "nt" else "clear")
    print("=====================================================")
    print("          Terima Kasih Atas Kunjungan Anda           ")
    print("=====================================================")
    exit()

input_saldo = int(input("Masukkan Saldo : "))

print("=====================================================")
print(f"NAMA ANDA              : {nama}")
print(f"SALDO ANDA SEKARANG    : Rp {input_saldo}")
print("=====================================================\n")

print("===================== MENU ATM ======================")
print("1. CEK SALDO")
print("2. TARIK TUNAI")
print("3. SETOR TUNAI")
print("4. TRANSFER")
print("5. KELUAR")
print("=====================================================")

menu = int(input("Pilih Menu : "))

os.system("cls" if os.name == "nt" else "clear")

if menu == 1:
    print("=====================================================")
    print(f"Saldo Anda Saat Ini: Rp {input_saldo}")
    print("=====================================================")

elif menu == 2:
    print("Saldo Anda Saat Ini:", input_saldo)
    tarik = int(input("Masukkan Jumlah Tarik Tunai: "))
    print("=====================================================")
    print("Saldo Anda Saat Ini: Rp", input_saldo - tarik)
    print("=====================================================")

elif menu == 3:
    print("Saldo Anda Saat Ini:", input_saldo)
    setor = int(input("Masukkan Jumlah Setor Tunai: "))
    print("=====================================================")
    print("Saldo Anda Saat Ini: Rp", input_saldo + setor)
    print("=====================================================")

elif menu == 4:
    print("Saldo Anda Saat Ini:", input_saldo)
    tf = int(input("Masukkan Jumlah Transfer: "))
    print("=====================================================")
    print("Saldo Anda Saat Ini: Rp", input_saldo - tf)
    print("=====================================================")

elif menu == 5:
    print("=====================================================")
    print("          Terima Kasih Atas Kunjungan Anda           ")
    print("=====================================================")

else:
    print("=====================================================")
    print("Menu Tidak Tersedia")
    print("=====================================================")